modules/nw-ingress-aws-static-eip-nlb-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-aws-static-eip-nlb-configuration_{context}"]
+= Configuring AWS Elastic IP (EIP) addresses for a Network Load Balancer (NLB)
+
+You can specify static IPs, otherwise known as elastic IPs, for your network load balancer (NLB) in the Ingress Controller. This is useful in situations where you want to configure appropriate firewall rules for your cluster network.
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your `IngressController`.
+
+.Procedure
+
+. Create a custom resource (CR) file.
++
+Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name> <1>
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    loadBalancer:
+      scope: External <3>
+      type: LoadBalancerService
+      providerParameters:
+        type: AWS
+        aws:
+          type: NLB
+          networkLoadBalancer:
+            subnets: <4>
+              ids:
+              - <subnet_ID>
+              names:
+              - <subnet_A>
+              - <subnet_B>
+            eipAllocations: <5>
+            - <eipalloc_A>
+            - <eipalloc_B>
+            - <eipalloc_C>
+----
+<1> Replace `<name>` with a name for the Ingress Controller.
+<2> Replace `<domain>` with the DNS name serviced by the Ingress Controller.
+<3> The scope must be set to `External` and be Internet-facing in order to allocate EIPs.
+<4> Specify the IDs and names for your subnets. The summation of IDs and names must be equal to your allocated EIPs.
+<5> Specify the EIP addresses.
++
+[IMPORTANT]
+====
+You can specify a maximum of one subnet per availability zone. Only provide public subnets for external Ingress Controllers. You can associate one EIP address per subnet.
+====
+
+. Save and apply the CR file using the following command:
++
+[source,terminal]
+----
+$  oc apply -f sample-ingress.yaml
+----
+
+.Verification
+
+. Confirm the load balancer was provisioned successfully by checking the `IngressController` conditions with the following command:
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command

Here, too. :)